soccerdata/sofascore.py

```diff
@@ -0,0 +1,265 @@
+"""Scraper for https://www.sofascore.com/."""
+
+import datetime
+import itertools
+import json
+from pathlib import Path
+from typing import Callable, Dict, Iterable, List, Optional, Union
+
+import pandas as pd
+
+from ._common import BaseRequestsReader, make_game_id
+from ._config import DATA_DIR, NOCACHE, NOSTORE, TEAMNAME_REPLACEMENTS
+
+SOFASCORE_DATADIR = DATA_DIR / 'Sofascore'
+SOFASCORE_API = 'https://api.sofascore.com/api/v1/'
+
+BIG_FIVE_DICT = {
+    "Serie A": "ITA-Serie A",
+    "Ligue 1": "FRA-Ligue 1",
+    "LaLiga": "ESP-La Liga",
+    "Premier League": "ENG-Premier League",
+    "Bundesliga": "GER-Bundesliga",
+}
```

Comment:
This was copy-pasted from the FBref scraper. I assume it is not needed for Sofascore.

Reply:
Yes, you are correct. I have excluded this now